### PR TITLE
Updated brazilian portugues translation

### PR DESCRIPTION
**Pull request by Felipefpl**
Updated the file with the newest strings and changed the year editing the file on github (poedit doesnt allow me to change the year of copyright) from 2019 to 2022.

### CODE REVIEW

File: language/Portuguese_Brazil/strings.po [hunk 1]
```diff
@@ -2988,7 +2992,7 @@ msgstr "Remover dos Favoritos"
 #, c-format
 msgctxt "menu-main"
 msgid "About %1$s"
-msgstr "Sobre o %1$s"
+msgstr "Sobre o %s"
```
Comment (cuavas) [inline language/Portuguese_Brazil/strings.po:2995]: This appears to be an inadvertent change.

File: language/Portuguese_Brazil/strings.po [hunk 2]
```diff
@@ -3142,7 +3146,7 @@ msgstr "Suportado: Sim"
 
 #: src/frontend/mame/ui/selmenu.cpp:681
 msgid "System is parent"
-msgstr "O sistema é pai"
+msgstr "O software é pai"
```
Comment (cuavas) [inline language/Portuguese_Brazil/strings.po:3149]: This appears to be an error – systems and software have separate messages.

File: language/Portuguese_Brazil/strings.po [hunk 3]
```diff
@@ -6523,17 +6527,1818 @@ msgstr ""
 "Insira a combinação ou pressione %s pra cancelar\n"
 "%s"
 
+#, c-format
+#~ msgid "System: \"%1$s\" software list "
+#~ msgstr "Sistema: lista dos softwares \"%1$s\" "
```
Comment (cuavas) [inline language/Portuguese_Brazil/strings.po:6532]: From here down, many obsolete message translations have been added.

**Review by cuavas — COMMENTED** [review]
Something seems to have gone wrong when you were editing the file.  There’s are a couple of errors, and many commented obsolete translations have been added to the bottom of the file.  I applied the newly translated messages in 57f6366ccb24fab9d4cd9db59fae314f15076672.  Thanks for the update.

**Comment by Felipefpl** [comment]
Then something is wrong with the english file hosted here, i update my .po file with the english file from this repository.

**Comment by cuavas** [comment]
Look at the diff preview on this pull request – your change adds a large number of commented obsolete translations to the end of the file.  It may have something to do with the way you’re using Poedit.  Sometimes Poedit behaves strangely.  That still doesn’t explain the change from “sistema” to “software” in the translation of “System is parent”.

**Pull request closed without merging**